ssl/ssl_lib.c

```diff
@@ -4361,7 +4361,8 @@ int ssl_validate_ct(SSL *s)
     CT_POLICY_EVAL_CTX_set1_cert(ctx, cert);
     CT_POLICY_EVAL_CTX_set1_issuer(ctx, issuer);
     CT_POLICY_EVAL_CTX_set_shared_CTLOG_STORE(ctx, s->ctx->ctlog_store);
-    CT_POLICY_EVAL_CTX_set_time(ctx, SSL_SESSION_get_time(SSL_get0_session(s)));
+    CT_POLICY_EVAL_CTX_set_time(
+            ctx, SSL_SESSION_get_time(SSL_get0_session(s)) * 1000);
```

Comment:
Would fail on 32-bit platform, because it will first perform 32-bit multiplication and only then convert to uint64_t (first extending sign bit of multiplication result by the way).